README.md

```diff
@@ -28,17 +28,25 @@
 
 ## Usage
 
-cf download APP_NAME [PATH] [--overwrite] [--verbose] [--omit omitted_path] [-i instance]
+cf download APP_NAME [PATH...] [--overwrite] [--file] [--verbose] [--omit omitted_path] [-i instance]
 
-The downloaded app files will put put in a new directory "APP_NAME-download" that's created within your working directory.
+The downloaded app files will be put in a new directory "APP_NAME" that's created within your working directory.
```

Comment:
If the entire application is being downloaded, yes. If a specific file or directory is being downloaded it will be placed in the current working directory.